src/containers/BooksList.js

```diff
@@ -21,30 +21,26 @@ const BookList = () => {
     : books;
 
   return (
-    <div>
-      <h1>Books List</h1>
-      <h3 className="filter-title">Filter by:</h3>
-      <CategoryFilter filter={handleCategoryFilter} />
-      <table>
-        <thead>
-          <tr>
-            <th>Book ID</th>
-            <th>Title</th>
-            <th>Category</th>
-          </tr>
-        </thead>
-        <tbody>
-          {
-            selectedCat.map((b) => (
-              <Book
-                item={b}
-                removeBook={handleRemoveBook}
-                key={b.id}
-              />
-            ))
-          }
-        </tbody>
-      </table>
+    <div className="BooksList">
+      <header className="flex">
+        <div className="header-leftside flex">
+          <h1>Bookstore CMS</h1>
+          <span>BOOKS</span>
+          <CategoryFilter filter={handleCategoryFilter} />
+        </div>
+        {/* <img src={userIcon} alt="user-icon" /> */}
+      </header>
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.